mvpa2/measures/rsa.py

```diff
@@ -22,6 +22,9 @@
     from scipy.spatial.distance import pdist, squareform
     from scipy.stats import rankdata, pearsonr
 
+from sklearn.linear_model import Lasso, Ridge
```

Comment:
Does this need a check to see if sklearn is available as an external? (Strangely enough, the travis tests seem to pass without this check...)